damus/Views/Profile/ProfileView.swift

```diff
@@ -429,6 +432,17 @@ struct ProfileView: View {
         .padding(.horizontal)
     }
 
+    var tabs: [(String, FilterState)] {
+        var tabs = [
+            (NSLocalizedString("Notes", comment: "Label for filter for seeing only notes (instead of notes and replies)."), FilterState.posts),
+            (NSLocalizedString("Notes & Replies", comment: "Label for filter for seeing notes and replies (instead of only notes)."), FilterState.posts_and_replies)
+        ]
+        if profile.pubkey != damus_state.pubkey {
```

Comment:
Soliciting feedback: should the Conversations tab be hidden if there are no conversations?

Reply:
I decided to hide the tab if there's no conversations so that it's a better user experience -- reduces unnecessary taps.